week-03/main.js

```diff
@@ -25,12 +34,26 @@ var list = [1, 2, 3, 4, 5];
 
 // sayHello('Mer') // "Hello Mer!"
 
+function sayHello(name) {
+  return 'Hello ' + name + '!';
+}
+
 // Define a function `createRange` that takes `min` and `max` as parameters and returns an array with all the numbers
 // between min and max
 // Example:
 
 // createRange(4, 8) // [4, 5, 6, 7, 8]
 
+// I know this isn't right but I can't get this one
+
+function createRange(min, max){
+  for (i=0; i<array.length; i++){
```

Comment:
`array` is undefined. This function should return a new array. Where would you create that array?